bigframes/ml/llm.py

```diff
@@ -103,7 +104,13 @@
     "You should use this model name only if you are sure that it is supported in BigQuery."
 )
 
+_REMOVE_DEFAULT_MODEL_WARNING = "Due to the frequent deprecation of LLM models, the default model will be removed in BigFrames 3.0."
```

Comment:
We can rephrase this a bit to put less blame on the Vertex team. Perhaps:

```suggestion
_REMOVE_DEFAULT_MODEL_WARNING = "Since upgrading the default model can cause unintended breakages, the default model will be removed in BigFrames 3.0. Please supply an explicit model to avoid this message."
```